README.md

```diff
@@ -28,10 +21,7 @@ To bring up the services for the application, do this once the images have finis
 bin/up.sh
 ```
 
-To add data to the event_planning database for the first time, use this command:
-```bashc
-psql -d event_planning -f ./server/go_events.sql -h 0.0.0.0
-```
+Bringing the services up will also load the data into the Postgres database. In production, the data is stored in an AWS RDS instance. The database service brought up by Docker Compose is just for local development purposes.
```

Comment:
I'm thinking to add functionality here that will set the dev environment, and load up a local postgres string. Would be useful for our local debugging